### ISSUE TITLE
Bug 54110592: [React Native Gallery]: All the content i.e., headers and static text is not visible in high contrast aquatic and desert themes.

### ISSUE THREAD

**Issue by Yajur-Grover** [microsoft/react-native-gallery#491]
**Build Details:**
OS Version: Dev (OS Build 27695.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.75.0
AT: High Contrast 
 

**Pre-Requisite:**  

Turn on High Contrast Aquatic Theme (Settings->Accessibility->Contrast themes->Aquatic/Dessert theme->Apply)

**Repro-Steps:**
Launch the React Native Gallery app.
Navigate to hamburger button and expand it.
Now navigate to any button and select it.
Now observe the issue in high contrast themes.
Actual Result:
All the group's headers and some text present in the whole application is not visible in high contrast aquatic and desert theme.

**Expected Result:**
All the content i.e., headers and static text should be visible clearly in high contrast aquatic and desert theme.

**User impact:**

Users with low vision will not be able to consume the text if it is not visible in High Contrast Themes.

Link to ADO bug: https://microsoft.visualstudio.com/OS/_workitems/edit/54110592
